presto-docs/src/main/sphinx/release/release-0.178.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.178
+=============
+
+General Changes
+---------------
+
+* Fix various memory accounting bugs, which reduces the likelihood of full GCs/OOMs
+* Fix a regression that causes queries that use the keyword "stats"
+  to fail to parse.
+* Fix an issue where a query does not get cleaned up on the coordinator after query failure.
+* Add ability to cast from `REAL`/`TINYINT`/`SMALLINT` to JSON
+* Add support for :func:`GROUPING`.
+* Add support for correlated subqueries in ``IN`` predicates.
+* Add :func:`to_ieee754_32` and :func:`to_ieee754_64` functions.
+
+SPI Changes
+-----------
+
+* Make environment available to resource group configuration managers.
+* Add more performance statistics to query completion event.
+
+Hive Changes
+------------
+
+* Fix high CPU usage due to schema caching when reading Avro files.
+* Preserve decompression error causes when decoding ORC files.
+
+In-Memory Connector Changes
```

Comment:
The connector is just called "Memory" in the documentation. (Having "Connector" here violates the rule of not putting "Connector" in release note headers, but it would be rather ambiguous otherwise.)